_includes/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="usa-grid">
+  <div class="usa-site-footer-content usa-width-one-half">
```

Comment:
Correct, `usa-site-footer` is just for the styleguide, that's why I added "site". 

I think we are going to have special styles just for the styleguide so we should agree on a naming convention to identify that. The class they'll need to look at is when they're looking at the footer component and see `usa-footer`.

Do you have another suggestion you think we should use?